aten/src/ATen/native/native_functions.yaml

```diff
@@ -7379,6 +7379,32 @@
   dispatch:
     DefaultBackend: smooth_l1_loss_backward
 
+- func: huber_loss.out(Tensor self, Tensor target, int reduction=Mean, float beta=1.0, *, Tensor(a!) out) -> Tensor(a!)
+  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
+  python_module: nn
+  dispatch:
+    CPU: huber_loss_out
+    CUDA: huber_loss_out
+
+- func: huber_loss(Tensor self, Tensor target, int reduction=Mean, float beta=1.0) -> Tensor
+  use_c10_dispatcher: full
+  python_module: nn
+  dispatch:
+    CPU, CUDA: huber_loss
+
+- func: huber_loss_backward.grad_input(Tensor grad_output, Tensor self, Tensor target, int reduction, float beta, *, Tensor(a!) grad_input) -> Tensor(a!)
```

Comment:
I think we can avoid registering these backward functions in native_functions.yaml, right @albanD?